plugins/scaffolder-backend/src/scaffolder/templater/cookiecutter.ts

```diff
@@ -16,17 +16,47 @@ import { TemplaterBase, TemplaterRunOptions } from '.';
  * limitations under the License.
  */
 import fs from 'fs-extra';
+import { JsonValue } from '@backstage/config';
+import { runDockerContainer } from './helpers';
 
 export class CookieCutter implements TemplaterBase {
+  private async fetchTemplateCookieCutter(
+    directory: string,
+  ): Promise<Record<string, JsonValue>> {
+    try {
+      return await fs.readJSON(`${directory}/cookiecutter.json`);
+    } catch (ex) {
```

Comment:
Probable check that this is an expected error? I assume ENOENT is alright but nothing else

Reply:
Yep 👍